RELEASE.md

```diff
@@ -39,8 +39,10 @@
 
 ## Major features and improvements
 * Added execution time to pipeline completion log.
+
```

Comment:
Can't comment on the correct line, but let's edit the header "Release 1.0.0" to "[PRE-RELEASE] 1.0.0" to indicate more clearly this isn't final.